src/Website/Views/Shared/_Header.cshtml

```diff
@@ -16,7 +16,7 @@
 			<ul class="nav navbar-nav navbar-right">
 				<li><a href="#">Login</a></li>
 				<li><a href="#about">Register</a></li>
-				<li><a href="#contact">Contact Us</a></li>
+				<li><a href="ContactUs">Contact Us</a></li>
```

Comment:
I see that this was udpated at some point to use the UrlHelper.  I've updated that code to point at the about us controller and appropriate action.